oarepo_ui/theme/assets/semantic-ui/less/oarepo_ui/themes/default/elements/button.overrides

```diff
@@ -3,4 +3,19 @@
     background: none;
     padding: 5px;
   }
-}
\ No newline at end of file
+}
+
+.ui.button {
+  &.clear-filters-button {
+    color: black;
```

Comment:
I wouldn't enforce black here, every site has different button color schemes

Reply:
@mirekys Hey, the issue is, that, when you let semantic handle the colors, you end up with this weak contrast level
![image](https://github.com/oarepo/oarepo-ui/assets/34883098/b82751d6-fd6b-4972-8a5e-ba3138dc3144)
And I honestly, don't see another way to do it, but to just change the text color to the one that gives a good contrast value. This was done in all three places for this reason. Should I just leave the default value instead please? For me it is a bit difficult to read the text.

Reply:
I now get the point here, you can leave it as is :)